src/server/graphql/models/Invitation/inviteTeamMembers/asyncInviteTeam.js

```diff
@@ -5,15 +5,16 @@ import resolveSentEmails from './resolveSentEmails';
 import makeInviteToken from './makeInviteToken';
 import makeInvitationsForDB from './makeInvitationsForDB';
 
-export default async function asyncInviteTeam (inviterUserId, teamId, invitees) {
+export default async function asyncInviteTeam(inviterUserId, teamId, invitees, unitTestCb) {
```

Comment:
Because many of our graphql endpoints do critical work in `setTimeout`, I need a mechanism to tell when this work is done or the `avajs` process terminates before the database is updated. I've had to sprinkle callbacks throughout various endpoints to facilitate testing.